treeherder/seta/preseed.py

```diff
@@ -20,11 +58,20 @@ def load_preseed():
     The fields buildtype, testtype and platform can have * which makes ut match  all flavors of
     the * field. For example: (linux64, pgo, *) matches all Linux 64 pgo tests
     """
-
+    logger.debug("START")
+    # XXX: Caching of runnable jobs is not working very well use global variable for now
+    global REF_NAMES
+    if not REF_NAMES:
+        REF_NAMES = get_reference_data_names("mozilla-central", "taskcluster")
     if not JobPriority.objects.exists():
+        logger.debug("There's no JobPriority objects in the table")
         return
+
     preseed = preseed_data()
     for job in preseed:
+        validate_preseed_entry(job)
```

Comment:
All of this PR is simply to validate all entries of `preseed.json` in here.
We should move this kind of logic into Travis so we fail there rather than on deployment.
We won't land this PR without that.